workflows/VGP-assembly-v2/Pre-curation-Hi-C-map-generation/PretextMap_Generation.ga

```diff
@@ -0,0 +1,2044 @@
+            "content_id": null,
+            "errors": null,
+            "id": 4,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "Suffix added to contigs belonging to haplotype 2. ",
+                    "name": "Haplotype 2 suffix"
+                }
+            ],
+            "label": "Haplotype 2 suffix",
+            "name": "Input parameter",
+            "outputs": [],
+            "position": {
+                "left": 78.57474659995361,
+                "top": 494.05478952156113
+            },
+            "tool_id": null,
+            "tool_state": "{\"default\": \"hap2\", \"parameter_type\": \"text\", \"optional\": true}",
+            "tool_version": null,
+            "type": "parameter_input",
+            "uuid": "294ab686-1b84-4a79-aa5c-10553af61911",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "5": {
+            "annotation": "",
+            "content_id": null,
+            "errors": null,
+            "id": 5,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "",
+                    "name": "HiC reads - forward"
+                }
+            ],
+            "label": "HiC reads - forward",
+            "name": "Input dataset collection",
+            "outputs": [],
+            "position": {
+                "left": 634.4517541662567,
+                "top": 1065.2398033276845
+            },
+            "tool_id": null,
+            "tool_state": "{\"optional\": false, \"tag\": null, \"collection_type\": \"list\"}",
+            "tool_version": null,
+            "type": "data_collection_input",
+            "uuid": "72de8338-3bca-41fa-89a2-d4093cfffd9b",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "6": {
+            "annotation": "",
+            "content_id": null,
+            "errors": null,
+            "id": 6,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "",
+                    "name": "HiC reads - reverse"
+                }
+            ],
+            "label": "HiC reads - reverse",
+            "name": "Input dataset collection",
+            "outputs": [],
+            "position": {
+                "left": 665.0496572173611,
+                "top": 1166.1304399701626
+            },
+            "tool_id": null,
+            "tool_state": "{\"optional\": false, \"tag\": null, \"collection_type\": \"list\"}",
+            "tool_version": null,
+            "type": "data_collection_input",
+            "uuid": "f697b004-cae2-4fd7-85a5-c37e5d67c11c",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "7": {
+            "annotation": "Trim 5 bases at the end of the paired reads. Use with some Arima hiC if the Hi-C map looks \"noisy\".",
```

Comment:
Is this really 5 bases or is this trimming bases from the 5' end ? If it's 5 bases from which end ? I'd also standardize on a spelling of Hi-C, I guess you've been using Hi-C mostly ?

Reply:
Yes 5 bases at the beginning of all reads

Reply:
But you said `5 bases at the end of the paired reads.`

Reply:
I changed to say at the beginning it was an error to say at the end